cellfinder_napari/detect.py

```diff
@@ -19,107 +27,50 @@
 
 
 def detect():
-    from math import ceil
-
-    # from fancylog import fancylog
-    # import cellfinder_napari as program_for_log
-    from napari.qt.threading import thread_worker
-    from cellfinder_core.main import main as cellfinder_run
-    from cellfinder_core.classify.cube_generator import get_cube_depth_min_max
-    from imlib.cells.cells import Cell
-    from .utils import cells_to_array
```

Comment:
glad you moved the imports 🤮

Reply:
I just saw on zulip, this has changed, so the imports can be moved to the proper place.